movo_robot/movo_bringup/launch/perception/movo_perception.launch

```diff
@@ -50,7 +50,7 @@
   <arg name="ordered_pc"                default="false"/>
 
   <arg name="enable_sync"               default="false"/>
-  <arg name="align_depth"               default="false"/>
```

Comment:
does align_depth refer to temporal alignment?

Reply:
It probably does align the messages in time to sync them but it mainly allows for the "aligned_depth_to_color" topic to be used in which the depth image is matched with the color image from the realsense